DEV_SETUP.md

```diff
@@ -64,33 +64,22 @@ NOTE: Developers on Mac OS have additional prerequisites. See the [Supplementary
     brew install python@3.9
     ```
 
-- A Python virtual environment manager like `pyenv` or `virtualenvwrapper`. We recommend `pyenv` ([see installation guide](https://github.com/pyenv/pyenv#installation))
-
-    To install python 3.9 with `pyenv`:
-
-    ```sh
-    pyenv install 3.9.11
-    ```
-
-    To set Python 3.9 as the global `python`, run:
-    ```sh
-    pyenv global 3.9.11
-    ```
-    Pro-tip: this is great for Mac OS users working around having to explicitly use `python3` :)
-
 
 - Requirements of Python libraries, if they aren't already installed.
 
   - **Linux**:
 
     ```sh
-    sudo apt install libncurses-dev libxml2-dev libxmlsec1-dev libxmlsec1-openssl libxslt1-dev libpq-dev pkg-config gettext
+    sudo apt-get update; sudo apt install libncurses-dev libxml2-dev libxmlsec1-dev \
+    libxmlsec1-openssl libxslt1-dev libpq-dev pkg-config gettext make build-essential \
+    libssl-dev zlib1g-dev libbz2-dev libreadline-dev libsqlite3-dev wget curl llvm \
```

Comment:
why `wget` and `curl`?

Reply:
Seems like at least some of this came from https://github.com/pyenv/pyenv/wiki#suggested-build-environment - could be useful to include a link to that

Reply:
yep, I merged those recommendations with what was previously here, after running into cryptic failures installing requirements